platforms/ide/tooling-api/src/main/java/org/gradle/tooling/events/problems/ProblemAggregationDescriptor.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.tooling.events.problems;
+
+import org.gradle.api.Incubating;
+
+import java.util.List;
+
+/**
+ * Describes a problem summary
```

Comment:
Summary or aggregation?
Maybe we should elaborate on why and how aggregation works here and only reference it in the base descriptor.